cmd_test.go

```diff
@@ -0,0 +1,134 @@
+package wish
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/charmbracelet/ssh"
+	"github.com/charmbracelet/wish/testsession"
+)
+
```

Comment:
```suggestion

var _ tea.ExecCommand = &Cmd{}

```

Static assertion that this type does indeed implement the `tea.ExecCommand` interface.